src/sentry/api/endpoints/organization_integration_migrate_opsgenie.py

```diff
@@ -15,7 +15,7 @@
 @region_silo_endpoint
 class OrganizationIntegrationMigrateOpsgenieEndpoint(RegionOrganizationIntegrationBaseEndpoint):
     publish_status = {
-        "PUT": ApiPublishStatus.UNKNOWN,
+        "PUT": ApiPublishStatus.PRIVATE,
```

Comment:
we are not publishing this?

Reply:
I don't believe so according to the [spreadsheet](https://docs.google.com/spreadsheets/d/1EaGibOxnqJa0vVo53woJwDRqMnBFdAxaG6eNHPe7xek/edit#gid=0&range=A20:C20)